CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [2.2.0] - 2020-01-21
```

Comment:
Wondering if we should be bumping this to version 3.0.0 as per https://semver.org/ as it feels like we are removing API functionality? Or has functionality been maintained by your bumping the version of the sock-client?

Reply:
It's complicated so for complete safety moving to 3 would be prudent *but* because of the way this change has been implemented 99% of users of the library won't see any breaking at all.

Reply:
The bump of the sock-client version is a development dependency and is only pulled/used when developing on *this* library. A user of the library will only be using v1 of the sock client if they have asked for it explicitly or via another dependency.

Reply:
Ah well explained. I think I am much clearer now. Thanks

Shall we go on the safe side and do the major bump to 3.0.0? I'm happy to follow whichever option you think is best.